### ISSUE TITLE
Alternative software to know about: Webcomic Reader

### ISSUE THREAD

**Issue by roryokane** [yumaikas/comic-trakr#2]
I haven’t tried this software, but from the description, it sounds like [Webcomic Reader](https://github.com/anka-213/webcomic_reader) might already have the features you were planning to add. Webcomic Reader is a user script, which is a type of simple browser extension that runs on top of another browser extension such as [Violentmonkey](https://violentmonkey.github.io/). Webcomic Reader has many features that make reading webcomics easier, but the relevant one is the “Remember this page” button, which adds your current page to a list.

<img width="542" alt="Screen Shot 2019-11-08 at 9 02 52 PM" src="https://user-images.githubusercontent.com/79168/68521072-5a6bec80-026b-11ea-892d-f8af9f95ec0d.png">

I think Webcomic Reader’s list is preserved in the extension’s local storage, so your software may be better if you find yourself wanting to keep your place even after switching which browser you are reading the comic in.

**Comment by yumaikas** [comment]
Yeah, that is good to know about. I don't usually use userscripts these days, mostly for security reasons. Right now, this uses a cookie, so it's no more effective than a userscript, and even less so. 

That being said, I would like to switch this to using localstorage, and/or add an "export my progress" page, to help move across browsers. 

What brought you to this repo?

**Comment by roryokane** [comment]
> What brought you to this repo?

1. [Best of Lobsters Fall 2019 | Lobsters](https://lobste.rs/s/e196fc/best_lobsters_fall_2019)
1. https://feed.junglecoder.com/view/bytag/BEST-OF
1. https://feed.junglecoder.com/
1. https://feed.junglecoder.com/view/idea/26
